### ISSUE TITLE
../lib/inspect_badfuncs.c:33: allowed_symbol: Assertion `symbol != NULL' failed

### ISSUE THREAD

**Issue by omajid** [rpminspect/rpminspect#1576]
**Describe the bug**

```
rpminspect: ../lib/inspect_badfuncs.c:33: allowed_symbol: Assertion `symbol != NULL' failed.
```

**To Reproduce**
Include the rpminspect command you ran including the NVRs of the builds passed to rpminspect.  Do not include a link to a Fedora, CentOS, or RHEL system integrating rpminspect.  If you are reporting a bug based on one of these, go to the *diagnostics* section of the rpminspect output and copy the rpminspect command string that was run.

No diagnostic command is visible in the web UI: https://artifacts.dev.testing-farm.io/ee49a5f8-46c6-4ec0-bb02-cbff719444a9/

https://artifacts.dev.testing-farm.io/ee49a5f8-46c6-4ec0-bb02-cbff719444a9/work-rpminspect06ebzy7s/rpminspect/execute/data/guest/default-0/rpminspect-1/data/verbose.log shows:

```
+ grep rpminspect
librpminspect-2.1-1.1.202511121936gita57fe527.x86_64
rpminspect-2.1-1.1.202511121936gita57fe527.x86_64
rpminspect-data-fedora-1.16-4.fc44.noarch
+ /usr/bin/rpminspect -c /usr/share/rpminspect/fedora.yaml --workdir /var/ARTIFACTS/work-rpminspect06ebzy7s/rpminspect/discover/default-0/tests/workdir/ --format=json --output=/var/ARTIFACTS/work-rpminspect06ebzy7s/rpminspect/execute/data/guest/default-0/rpminspect-1/data/result.json --verbose --release=fc43 dotnet10.0-10.0.106-1.fc43 dotnet10.0-10.0.107-1.fc43
...
Running abidiff inspection...        pass
Running addedfiles inspection...     pass
Running annocheck inspection...      FAIL
Running arch inspection...           pass
rpminspect: ../lib/inspect_badfuncs.c:33: allowed_symbol: Assertion `symbol != NULL' failed.
```

**Expected behavior**
I get valid results 

**Environment:**
- This is Fedora's CI

**Comment by dcantrell** [comment]
I think this was due to a double free.  I found and fixed it yesterday and the latest Copr builds for rpminspect have the fix.  I am unable to reproduce this issue locally now.

**Comment by omajid** [comment]
Thanks!
